src/libraries/oob.proj

```diff
@@ -18,6 +18,7 @@
 
   <ItemGroup Condition="'$(BuildTargetFramework)' != '$(NetCoreAppCurrent)' or '$(BuildTargetFramework)' == ''">
     <ProjectReference Include="oob-all.proj" SkipGetTargetFrameworkProperties="true" />
+    <ProjectReference Include="oob-gen.proj" />
```

Comment:
nit: this references all library generators, not just OOB ones.  You could name it `all-gen.proj`.

Reply:
It's the same with "oob-all.proj" which also references everything. I felt that it's better to follow the existing naming schema.